_board/swan.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: download
+board_id: "swan"
+title: "Swan Download"
+name: "Swan"
+manufacturer: "Blues Wireless"
+board_url: "https://blues.io/products/swan"
+board_image: "swan.jpg"
+date_added: 2021-9-29
+features:
+  - Feather-Compatible
+  - Battery Charging
+  - Breadboard-Friendly
+---
+
+Swan is a low-cost embeddable STM32L4-based microcontroller designed to accelerate the development and deployment of battery-powered solutions. It is especially useful for applications requiring large memory or a high degree of I/O expandability at an affordable cost, such as edge inferencing and remote monitoring.
+
+Uniquely for Feather-compatible boards, Swan is designed to satisfy developers’ needs that span from early prototyping through high-volume deployment. Developers may begin to use Swan in conjunction with Adafruit’s myriad sensors and FeatherWing-compatible carriers. Due to its novel design, for high-volume deployment the low-cost Swan can also be soldered directly to a parent PCB integrating those sensors, utilizing the full range of Swan’s I/O capabilities.
+
+The board has three independent power options – USB, Battery, or Line power – and provides a software-switchable 2 Amp regulator for powering external sensors. When operating in its low-power operating mode, the entire Swan board commonly draws only about 8uA while retaining all of its memory, making it quite suitable for battery-powered devices.
```

Comment:
I wouldn't capitalize battery and line

Reply:
also, I don't like "commonly draws only about"..."commonly" = maybe it does! "about" = somewhere in this range. I'd prefer we just said "draws ~8uA"